WindowsServerDocs/security/tls/what-s-new-in-tls-ssl-schannel-ssp-overview.md

```diff
@@ -62,6 +62,8 @@ In Windows Server 2012 and Windows 8, changes were made to the underlying authen
 
 -   Compatibility to previous versions of Windows operating systems is preserved.
 
+Also note that if System Mapper is enabled by the client application and you have configured SendTrustedIssuers that system mapper will add CN=NT Authority to the issuers list.

```

Comment:
Suggestion for using a MarkDown Note blob (colored) to highlight the note text:
```suggestion
> [!NOTE]
> If System Mapper is enabled by the client application and you have configured `SendTrustedIssuers`, that system mapper will add `CN=NT Authority` to the issuers list.

```
I have also added MarkDown back ticks around the phrases "SendTrustedIssuers" and "CN=NT Authority", both to keep Machine Translation from automatically translating part of the text and also to slightly emphasize the phrases as technical terms.

You are of course free to modify my suggestion before committing it to your PR branch.